cloud/azure/go.mod

```diff
@@ -3,48 +3,90 @@ module github.com/nitrictech/nitric/cloud/azure
 go 1.19
 
 require (
-	github.com/Azure/azure-sdk-for-go v56.3.0+incompatible
-	github.com/Azure/azure-storage-blob-go v0.13.0
+	github.com/Azure/azure-sdk-for-go v67.1.0+incompatible
+	github.com/Azure/azure-storage-blob-go v0.15.0
 	github.com/Azure/azure-storage-queue-go v0.0.0-20191125232315-636801874cdd
-	github.com/Azure/go-autorest/autorest v0.11.18
-	github.com/Azure/go-autorest/autorest/adal v0.9.14
+	github.com/Azure/go-autorest/autorest v0.11.28
+	github.com/Azure/go-autorest/autorest/adal v0.9.21
 	github.com/Azure/go-autorest/autorest/azure/auth v0.5.8
 	github.com/Azure/go-autorest/autorest/date v0.3.0
+	github.com/getkin/kin-openapi v0.112.0
 	github.com/golang/mock v1.6.0
 	github.com/golangci/golangci-lint v1.50.1
 	github.com/google/addlicense v1.1.0
 	github.com/mitchellh/mapstructure v1.5.0
+	github.com/nitrictech/cli v1.14.3
```

Comment:
Remove dependency on nitric cli. I think this is mostly the common tags stuff which should now be available in cloud/common.